src/std/os/signal.ss

```diff
@@ -22,7 +30,12 @@ package: std/os
   sigaddset
   sigdelset
   sigismember
-)
+  
+  SIG_BLOCK SIG_UNBLOCK SIG_SETMASK)
+
+(cond-expand
+  (openbsd
+    (def SIGMAX 32)))
```

Comment:
Can you also define it for `linux` as `SIGRTMAX + 1`?
And then can you change `signal-handler.ss` as well to use it for the signal validity test in `add-signal-handler!/remove-signal-handler!`? The test should be exclusive with these definitions, not inclusive.